lib/alerts_viewer_web/components/core_components.ex

```diff
@@ -636,10 +636,16 @@ defmodule AlertsViewerWeb.CoreComponents do
   end
 
   @doc """
-  Renders a date-time using human-friendly formatting.
+  Renders a date-time in Eastern time using human-friendly formatting.
   """
   attr(:date_time, :map, required: true)
 
+  def date_time(%{date_time: %DateTime{}} = assigns) do
```

Comment:
question: Why the new function definition rather than just updating the existing function?

Reply:
A lot of them come in as `nil` and throw an error if you convert them.